pmm-app/src/pmm-qan-app-panel/QueryAnalytics/Filters/Filters.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { ReactElement, useContext, useEffect, useState } from 'react';
+import { Humanize } from '../../../react-plugins-deps/components/helpers/Humanize';
+import './Filters.scss';
+import { Divider } from 'antd';
+import { StateContext } from '../StateContext';
+import FiltersService from './Filters.service';
+import { CheckboxField } from '../../../react-plugins-deps/components/FormComponents/Checkbox/Checkbox';
+import { useForm } from 'react-final-form-hooks';
+import { Form as FormFinal } from 'react-final-form';
+import Search from 'antd/es/input/Search';
+
+const checkboxGroup = (form, name, items, group, showAll, filter, labels) => {
```

Comment:
This is a separate component, please refactor so that we stick to the `one component per file` rule